api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/json/GetRandomSearchIndividualsByVClass.java

```diff
@@ -75,7 +75,10 @@ private String renderShortView(String individualUri, String vclassName) {
 		modelMap.put("individual",
 				IndividualTemplateModelBuilder.build(individual, vreq));
 		modelMap.put("vclass", vclassName);
-		String langCtx = vreq.getLocale().getLanguage() + "-"+vreq.getLocale().getCountry();  //UQAM-Linguistic-Management build the linguistic context
+		String langCtx = vreq.getLocale().getLanguage();  //UQAM-Linguistic-Management build the linguistic context
+        if (!vreq.getLocale().getCountry().isEmpty()) {
```

Comment:
The formatting in the VIVO codebase is... inconsistent. However, it is helpful not to exacerbate the chaos. Would you mind changing the indentation of the lines you added to either use spaces or tabs in order to match the lines around it?

Reply:
understandable, i changed it.